### PR TITLE
Remove support for `input` elements

### LINKED ISSUE #6: Drop support for `input`
Firefox's UA styles fix the line-height of `input` to `normal`. Can't remember the last time I've seen `input type="button|submit"` used instead of `button`, so the component could just depend on using `button` or `a` and be a bit more flexible.

Could change to using `line-height: inherit`.

### PR DESCRIPTION
**Pull request by AntonTrollback**
Fixes #6

### CODE REVIEW

**Comment by simonsmith** [comment]
I know it's ancient but I'd vote to merge this, @timkelty @giuseppeg?

File: README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Read more about [SUIT's design principles](https://github.com/suitcss/suit/).
 * `Button` - [core] The core button component
 * `is-disabled` - [state] For disabled-state button styles (themes)
 
-N.B. You must also include the `disabled` attribute on `button` and `input`
-elements. For `a` elements, you should remove the `href` attribute and prevent
-JavaScript event handlers from firing.
+N.B. You must also include the `disabled` attribute on `button` elements. For
+`a` elements, you should remove the `href` attribute and prevent JavaScript
```
Comment (giuseppeg) [inline README.md]: Why not use a `span` then? I try to advocate this as much as possible but it is a tough one to win :)

**Comment by giuseppeg** [comment]
@simonsmith sounds good to me

**Comment by timkelty** [comment]
I don't think we should be endorsing an `a` with no href (or a clickable `span`).

If it's clickable, it should be a `<button>` element. If its in a form and you don't want it to submit, you do `<button type="button">`.

**Comment by AntonTrollback** [comment]
+1 merge and drop the "a" line

**Comment by simonsmith** [comment]
Preventing default events is a valid use of JS I would say, but removing the `href` not so much.

**Comment by simonsmith** [comment]
Okay, will tidy this up and get it merged.

**Comment by simonsmith** [comment]
https://github.com/suitcss/components-button/compare/PR/11 Look good? I'm thinking a major version bump too

**Comment by timkelty** [comment]
:shipit:

**Comment by simonsmith** [comment]
Merged in https://github.com/suitcss/components-button/commit/7d8936c024a824c2501982a9c0beaf45ed976e5a

**Pull request closed without merging**